src/Bundle.js

```diff
@@ -17,12 +18,16 @@ export default class Bundle {
 		this.entry = options.entry;
 		this.entryModule = null;
 
-		this.resolveId = options.resolveId || defaultResolver;
-		this.load = options.load || defaultLoader;
+		this.resolveId = ensureArray( options.resolveId )
```

Comment:
using `first` (see below) instead of `attempt`, this'd presumably look something like this:

```js
this.resolveId = first( ensureArray( options.resolveId ).concat( defaultResolver ) );
```

(or the fallback could be a second argument, and the `first` function would do the concatting...)